elifetools/parseJATS.py

```diff
@@ -2852,7 +2852,7 @@ def author_email_addresses(author, correspondence):
                 email_addresses.append(affiliation["email"])
 
     # Also look at the author attributes
-    if author.get("corresp") and author.get("email"):
```

Comment:
Do you no longer need the `'corresp'` authors?

Reply:
It looks like this is one fewer place where `corresp` is considered in the logic. I thought we reported this value in Crossref and Pubmed deposits, which are more upstream users of the parsed data (the JSON output being the most recent incarnation of parsed output from this library) but I was surprised to see it isn't relied upon too much.

I'm not sure this is the right time to completely remove parsing the `corresp` value of `<contrib>` tags from JATS XML until we're completely sure no other projects or users of the general purpose parsing functions are satisfied we should not parse the value. I would err on having more data extracted from the XML than less.